packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -342,3 +305,46 @@ export class MonacoEditor implements TextEditor, IEditorReference {
     }
 
 }
+export namespace MonacoEditor {
+    export interface ICommonOptions {
+        /**
+         * Whether an editor should be auto resized on a content change.
+         *
+         * #### Fixme
+         * remove when https://github.com/Microsoft/monaco-editor/issues/103 is resolved
+         */
+        autoSizing?: boolean;
+        /**
+         * A minimal height of an editor.
+         *
+         * #### Fixme
+         * remove when https://github.com/Microsoft/monaco-editor/issues/103 is resolved
+         */
+        minHeight?: number;
+    }
+
+    export interface IOptions extends ICommonOptions, IEditorConstructionOptions { }
+
+    export function getAll(manager: EditorManager): MonacoEditor[] {
+        return manager.editors.map(e => get(e)).filter(e => !!e) as MonacoEditor[];
+    }
+
+    export function getCurrent(manager: EditorManager): MonacoEditor | undefined {
+        return get(manager.currentEditor);
+    }
+
+    export function getActive(manager: EditorManager): MonacoEditor | undefined {
+        return get(manager.activeEditor);
+    }
+
+    export function get(editorWidget: EditorWidget | undefined) {
+        if (editorWidget && editorWidget.editor instanceof MonacoEditor) {
+            return editorWidget.editor;
+        }
+        return undefined;
+    }
+
+    export function findByDocument(manager: EditorManager, document: MonacoEditorModel): MonacoEditor[] {
+        return getAll(manager).filter(editor => editor.documents.has(document));
```

Comment:
would the predicate `editor.document === document` work too?

Reply:
no, diff editor has 2 documents

Reply:
thanks! I got it. the other is added in super constructor.